src/sb_gazebo/urdf/jfrost.xacro

```diff
@@ -3,178 +3,194 @@
      The Model for jfrost, a 2-wheel drive robot with rear caster wheels
 -->
 <robot name="jfrost" xmlns:xacro="http://www.ros.org/wiki/xacro">
-<xacro:property name="PI" value="3.1415926535897931"/>
+  <xacro:property name="PI" value="3.1415926535897931"/>
 
-<xacro:property name="chassis_length" value="0.91" />
-<xacro:property name="chassis_width" value="0.71" />
-<xacro:property name="chassis_height" value="0.35" />
-<xacro:property name="chassis_mass" value="30" />
-<xacro:property name="chassis_elevation" value="0.1" />
+  <xacro:property name="chassis_length" value="0.91"/>
+  <xacro:property name="chassis_width" value="0.71"/>
+  <xacro:property name="chassis_height" value="0.35"/>
+  <xacro:property name="chassis_mass" value="30"/>
+  <xacro:property name="chassis_elevation" value="0.1"/>
 
-<xacro:property name="tower_height" value="0.9"/>
-<xacro:property name="tower_thickness" value="0.04"/>
-<xacro:property name="tower_mass" value="0"/>
+  <xacro:property name="tower_height" value="0.9"/>
+  <xacro:property name="tower_thickness" value="0.04"/>
+  <xacro:property name="tower_mass" value="0"/>
 
-<xacro:property name="wheel_radius" value="0.15" />
-<xacro:property name="wheel_width" value="0.15" />
-<xacro:property name="wheel_displace" value="0.25" />
-<xacro:property name="wheel_mass" value="1" />
+  <xacro:property name="wheel_radius" value="0.15"/>
+  <xacro:property name="wheel_width" value="0.15"/>
+  <xacro:property name="wheel_displace" value="0.25"/>
+  <xacro:property name="wheel_mass" value="1"/>
 
-<xacro:property name="camera_size" value="0.05" />
-<xacro:property name="camera_mass" value="0.1" />
-<xacro:property name="camera_angle" value="0.5235" />
+  <xacro:property name="camera_size" value="0.05"/>
+  <xacro:property name="camera_mass" value="0.1"/>
+  <xacro:property name="camera_angle" value="${pi/8}"/>
```

Comment:
Didn't see that oops, but: 
```
New in Jade:

Since ROS Jade, Xacro employs python to evaluate expressions enclosed in dollared-braces (${}). 
This allows for more complex arithmetic expressions. Also, some basic constants, e.g. pi, are already predefined:
```